src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/ServiceClient.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation.model.response.ClientServiceErrorResponse;
+
+import java.io.IOException;
+
+abstract class ServiceClient {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This abstract class does not have any abstract methods](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4508828)